pkg/ebpf/tracer.go

```diff
@@ -323,7 +327,15 @@ func (m *FlowFetcher) Close() error {
 		if err := m.objects.DirectFlows.Close(); err != nil {
 			errs = append(errs, err)
 		}
-		m.objects = nil
+		if err := m.objects.DnsFlows.Close(); err != nil {
+			errs = append(errs, err)
+		}
+		if err := m.objects.FlowSequences.Close(); err != nil {
+			errs = append(errs, err)
+		}
+		if len(errs) == 0 {
```

Comment:
why do we need to keep the objects reference if there are errors?

Reply:
in case I wanted to retry